backend/schemas.py

```
@@ -3,6 +3,7 @@
 from datetime import datetime
 from enum import Enum
 from uuid import UUID
+from pydantic import BaseModel
```

Ruff (0.14.8)

Redefinition of unused `BaseModel` from line 1

Remove definition: `BaseModel`

(F811)